build.gradle

```diff
@@ -38,3 +40,31 @@ dependencies {
 }
 
 defaultTasks 'shadowJar'
+
+publishing {
+    publications {
+        MyPublication(MavenPublication) {
+            from components.java
+            groupId group
+            artifactId 'shell'
+            version version
+        }
+    }
+}
+
+bintray {
+    user = System.getenv('BINTRAY_USER')
+    key = System.getenv('BINTRAY_KEY')
+    publications = ['MyPublication']
+    dryRun = false
+    publish = false
+    pkg {
+        repo = 'maven'
+        name = 'osiam-shell'
+        licenses = ['MIT']
+        vcsUrl = 'https://github.com/osiam/shell.git'
+        version {
+            name = version
```

Comment:
Sounds odd, doesn't it? If you have a look at [bintray's example](https://github.com/bintray/bintray-examples/blob/master/gradle-bintray-plugin-examples/publications-example/build.gradle#L64), they hardcode a version string there. 

It is the "name" of the version, that is at least my understanding.

I am not sure why they duplicate the version information so often, that is why I chose to re-use the existing variables. But it might well be that there is a reason that I did miss.